python/strings_udf/strings_udf/tests/test_cmpops.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
```

Comment:
I find it a little unusual that we put all the tests in separate files for every function... that feels like overkill since the files are so short. Maybe there are a small number of natural groupings, or just lump them all into one or two files.

Reply:
I merged these together.